src/app/content/components/BookBanner.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import withServices from '../../context/Services';
+import styled from 'styled-components';
+import theme from '../../theme';
+import { AppServices, AppState } from '../../types';
+import * as select from '../selectors';
+import { Book, Page } from '../types';
+import {ChevronLeft} from 'styled-icons/boxicons-regular/ChevronLeft'
+import { findArchiveTreeSection, bookDetailsUrl } from '../utils'
+import Color from 'color';
+import typography from '../../components/Typography';
+
+const LeftArrow = styled(ChevronLeft)`
+  height: 2rem;
+  width: 2rem;
+  color: #FFFFFF;
+  margin-right: 0.7rem;
+`;
+
+interface PropTypes {
+  page?: Page;
+  book?: Book;
+  services: AppServices;
+}
+
+const TopBar = styled.div`
+  width: 100%;
+  max-width: 117rem;
+  background: transparent;
+  margin: 0 auto;
+  text-align: left;
+`;
+
+const BookTitle = styled.a`
+  ${typography.h4Style}
+  letter-spacing: -0.04rem;
+  color: ${theme.color.primary.blue.foreground};
+  text-align: left;
+  display: flex;
+  align-items: center;
+  width: 100%;
+  max-width: 87rem;
+  margin: 0;
+  text-decoration: none;
+
+  @media (max-width: 700px) {
+    font-size: 1.6rem;
+    line-height: 2.5rem;
+  }
+`;
+
+const BookChapter = styled.h3`
```

Comment:
this needs ellipsis text overflow, but its actually kind of complicated for the book chapter because Devika wants it to ellipsis normally for desktop, but allow wrapping onto two lines before ellipsis on mobile, which is not actually possible using the text-overflow property. 

i haven't looked into it a lot, but it seems _possible_ if hacky http://hackingui.com/front-end/a-pure-css-solution-for-multiline-text-truncation/